### PR TITLE
implemented cksum

### PR DESCRIPTION
**Pull request by cospplredman**
quick little implementation of cksum ��

### CODE REVIEW

File: src/cksum/cksum
Comment (Sigmanificient) [inline src/cksum/cksum:1]: Please remove the binary and then ensure the binary is not pushed by adding it to the `.gitignore` list

File: src/cksum/cksum.c [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#include<stdint.h>
+#include<stdio.h>
+
+#define NAME "cksum (canoutils)"
+#define VERSION "0.0.1"
+#define AUTHOR "cospplredman"
+
+#include"../version_info.h"
```
Comment (Sigmanificient) [inline src/cksum/cksum.c:8]:
```suggestion
#include "/version_info.h"
```

File: src/cksum/cksum.c [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+			if(f==2){f=1;print_version();}
+			continue;
+		}
+		FILE*f=fopen(v[k],"r");
+		if(f==NULL)exit(1);
+		X(;(d=getc(f))>=0;t++,d)X(j=t;j;,(j&0xff),j>>=8)X(j=3;j--;,0)
+		if(printf("%u %d %s\n",~((r<<8)|(e&0xff)),t,v[k])<0)exit(1);
+		r=e=t=0;
+	}
+}
```
Comment (Sigmanificient) [inline src/cksum/cksum.c:26]: Can you refactor your code to make it readable? The goal isn't to code golf the coreutils

**Review by Sigmanificient — CHANGES_REQUESTED** [review]

File: src/cksum/cksum
Comment (cospplredman) [reply src/cksum/cksum:1]: �

**Comment by proh14** [comment]
red-birb-wuewue-ezgif com-speed

**Pull request closed without merging**